src/Kernel/AccessValidator/Requests/IAccessValidatorCRServiceRequest.cs

```diff
@@ -1,13 +1,13 @@
-﻿using System;
-
-namespace LT.DigitalOffice.Kernel.AccessValidator.Requests
-{
-    /// <summary>
-    /// Message request model that is sent to UserService and CheckRightsService via MassTransit.
-    /// </summary>
-    public interface IAccessValidatorRequest
-    {
-        Guid UserId { get; set; }
-        int RightId { get; set; }
-    }
-}
+﻿using System;
+
+namespace LT.DigitalOffice.Kernel.AccessValidator.Requests
+{
+    // <summary>
+    /// Message request model that is sent to CheckRightsService via MassTransit.
+    /// </summary>
+    public interface IAccessValidatorCRServiceRequest

```

Comment:
Perhaps it is worth writing a complete transcript.

Reply:
There is code have documentation

Reply:
= IAccessValidatorCRServiceRequest -> IAccessValidatorCheckRightsServiceRequest

Reply:
Fixed